### DISCUSSION TITLE
dodanie-klienta-torrent-do-routera-z-openwrt/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#100] [General]
# dodanie-klienta-torrent-do-routera-z-openwrt/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/dodanie-klienta-torrent-do-routera-z-openwrt/

**Comment by cookers-hi** [comment]
Mógłby Pan dodać poradnik jak dodać czytnik rss do transmission? ;)

**Reply by idarek** [reply]
Transmission sam w sobie nie obsługuje RSS. Aby dodać obsługę, trzeba dodać dodatkowy pakiet. Jako, że nie używam tego typu rozwiązania, na chwilę obecną nie będę się w to ingerował.
